docs/website/postprocess_dialect_docs.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# MLIR dialect markdown document postprocessor for website generation.
```

Comment:
I think it would be nice to a) make this a docstring instead of a comment because why not, and b) expand a *little* on why we're doing postprocessing at all